dataplane/standalone/port.cc

```diff
@@ -116,6 +126,36 @@ sai_status_t Port::create(_In_ uint32_t attr_count,
 }
 
 sai_status_t Port::set_attribute(_In_ const sai_attribute_t* attr) {
+  LOG(INFO) << "port set: " << attr->id;
+  switch (attr->id) {
+    case SAI_PORT_ATTR_ADMIN_STATE:
+      if (!this->portExists) {
+        return SAI_STATUS_SUCCESS;
+      }
+      grpc::ClientContext context;
+      forwarding::PortStateRequest req;
+      req.mutable_context_id()->set_id("lucius");
+      req.mutable_port_id()->mutable_object_id()->set_id(this->id);
+
+      if (attr->value.booldata) {
+        LOG(INFO) << "Setting to port up, id " << this->id;
+        req.mutable_operation()->set_admin_status(
+            forwarding::PORT_STATE_ENABLED_UP);
+      } else {
+        LOG(INFO) << "Setting to port down, id " << this->id;
+        req.mutable_operation()->set_admin_status(
+
+            forwarding::PORT_STATE_DISABLED_DOWN);
+      }
+
+      forwarding::PortStateReply resp;
+      auto status = this->fwd->PortState(&context, req, &resp);
+      if (!status.ok()) {
+        LOG(ERROR) << "Failed to port state: " << status.error_message();
+        return SAI_STATUS_FAILURE;
+      }
```

Comment:
This looks similar to the routine in hostif, maybe have a TODO to dedup.

Reply:
done